WcaOnRails/app/controllers/delegates_controller.rb

```diff
@@ -3,8 +3,8 @@
 class DelegatesController < ApplicationController
   before_action :authenticate_user!
   before_action -> { redirect_to_root_unless_user(:can_view_delegate_matters?) }, only: [:stats]
-  before_action -> { redirect_to root_url unless current_user_is_authorized? }, only: [:delegate_probation_data, :probations]
-  before_action :current_user_is_authorized_for_action!, only: [:delegate_probation_data, :start_delegate_probation, :end_delegate_probation]
+  before_action -> { redirect_to root_url unless current_user_is_authorized? }, only: [:probations]
```

Comment:
Please clean this up similar to what you did in the WFC panel fixup PR. This includes:
- creating a permission method in the `user.rb` model, that checks the same as `current_user_is_authorized?` below.
    - This should include a check whether the user is an admin. Admins are always allowed to do anything.
    - See whether you can reuse the existing `can_view_delegate_matters` or other methods. Only rewrite permissions that don't exist yet.
- Change this invocation (the line that is commented here on GH) to `redirect_to_root_unless_user` and use the newly created method
- Remove the `private def` block below